examples/financial-research-agent/agents.ts

```diff
@@ -69,9 +69,8 @@ Focus on key numbers, events, or quotes that will be useful to a financial analy
 export const searchAgent = new Agent({
   name: 'FinancialSearchAgent',
   instructions: searchAgentPrompt,
-  model: 'gpt-4.1',
+  model: 'gpt-5.2',
   tools: [webSearchTool()],
-  modelSettings: { toolChoice: 'required' },
```

Comment:
fyi, i've removed the modelsettings because tool_choice: auto is the default one.